src/citrine/resources/table_config.py

```diff
@@ -388,9 +416,11 @@ class TableConfigCollection(Collection[TableConfig]):
     # definition) are necessary
     _individual_key = None
 
-    def __init__(self, project_id: UUID, session: Session):
-        self.project_id = project_id
-        self.session: Session = session
+    def __init__(self, *args, team_id: UUID, project_id: UUID = None, session: Session = None):
+        args = _pad_positional_args(args, 2)
+        self.project_id = project_id or args[0]
+        self.session: Session = session or args[1]
+        self.team_id = team_id
```

Comment:
Why would a Table Config need a Project ID?  Are Tables still affiliated with particular Projects and not affiliated with datasets anymore?

Reply:
My understanding is that it's still needed to retrieve it based on the table it built: https://github.com/CitrineInformatics/citrine-python/blob/c7117a6640bd5be6bae97b09e29eb527bd02e734/src/citrine/resources/table_config.py#L460

I easily could be wrong though. @pacdaemon did I misinterpret what you said?